src/quicktype-core/language/ruby/index.ts

```diff
@@ -553,7 +555,9 @@ export class RubyRenderer extends ConvenienceRenderer {
 
     private emitTypesModule() {
         this.emitBlock(["module Types"], () => {
-            this.emitLine("include Dry::Types.module");
+            const includeType = this._options.legacyGemVesion ? "Dry::Types.module" : "Dry.Types(default: :nominal)";
+            const intTypeDeclaration = this._options.legacyGemVesion ? "Integer" : "Int";
```

Comment:
Also this might be a bug. Should probably be the other way around:

```suggestion
            const intTypeDeclaration = this._options.legacyGemVesion ? "Int" : "Integer";
```

Like there:

https://github.com/quicktype/quicktype/pull/1385/files#diff-94c47b5848b3bd8b852453de72cac7f5R163